apps/emqx_authz/src/emqx_authz_utils.erl

```diff
@@ -130,6 +132,25 @@ render_sql_params(ParamList, Values) ->
         #{return => rawlist, var_trans => fun handle_sql_var/2}
     ).
 
+-spec parse_http_resp_body(binary(), binary()) -> allow | deny | ignore | error.
+parse_http_resp_body(<<"application/x-www-form-urlencoded", _/binary>>, Body) ->
```

Comment:
This breaks backward compatibility?
We should also check if the body is empty, then fallback to the old behavior?
5.0.0 and 4.x